draft-pda-protocol.md

```diff
@@ -1104,6 +1099,8 @@ can defeat privacy by leaking input outside of the Prio system.
 1. The input validation protocol executed by the aggregators prevents either
 individual clients or coalitions of clients from compromising the robustness
 property.
+1. If aggregator output satisifes differential privacy {{dp}}, then all records
+not leaked by malicious clients are still protected.
```

Comment:
Would you mind adding a reference for `{{dp}}` and verify the document builds by running `make`? See the top of the document for guidance. If you don't have time, then please at least provide a link to an article you think would be a suitable reference. The most useful reference would be something that describes the distinction between client-side and server-side noise addition.

Reply:
Oops, I missed that {{dp}} refers to a section and not a paper. Disregard.